components/tools/OmeroPy/src/omero/plugins/admin.py

```diff
@@ -34,6 +34,7 @@
 
 from omero.plugins.prefs import \
     WriteableConfigControl, with_config, with_rw_config
+from omero.plugins.prefs import windows_warning, WINDOWS_WARNING
```

Comment:
I'd suggest we not cross-link all the plugins. This can lead to odd loading issues. If need me, let's create a single `OmeroPy/src/omero/install/windows_warning.py` file which can be imported.